code/game/objects/items/theft_items.dm

```diff
@@ -158,6 +158,22 @@
 	icon_state = "supermatter_sliver"
 	pulseicon = "supermatter_sliver_pulse"
 
+/obj/item/nuke_core/supermatter_sliver/process()
+	. = ..()
+	var/new_filter = isnull(get_filter("ray"))
+
+	add_filter(name = "ray", priority = 1, params = list(
+		type = "rays",
+		size = 40,
+		color = "#ffd04f",
+		factor = 6,
+		density = 20
+	))
```

Comment:
Do we need to call this every `process()`?

Reply:
Perhaps only on initialize, let me see, I presumed rays were dealt like this with the supermatter on TG for a reason, so we'll see what happens when it runs once.

Reply:
This is needed. Otherwise the rays do not move, if only the animate is left. Rays must be added each process, it seems?